apps/web/src/types/mux-player-react.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module '@mux/mux-player-react' {
+  export interface MuxPlayerProps {
+    [key: string]: any;
+    playbackId: string;
+    metadata?: Record<string, any>;
+  }
+
+  export default function MuxPlayer(props: MuxPlayerProps): JSX.Element
+}
```

Comment:
Why is this required? They don't have typings?

Reply:
Correct. That was my experience. I installed their packages and the linter couldn't find declarations. They also don't seem to have `@types/...` package.

Reply:
However, [their project](https://github.com/muxinc/elements/tree/main/packages/mux-player-react) is in TypeScript. It's weird that the linter is failing on it because it doesn't fail on any other package.

Reply:
Resolved. Seemed to be a cache issue.